bin/phpunit

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env php
```

Comment:
I'm not sure this is required as our tests are run from vendor/bin/simple-phpunit